src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +25,26 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws FileNotFoundException {
+        StringBuilder text = new StringBuilder();
+        try (Scanner scanner = new Scanner(new FileReader(fileName))) {
+            while (scanner.hasNextLine()) {
+                text.append(scanner.nextLine().toLowerCase().replaceAll("[^a-z]", ""));
+            }
```

Comment:
you can find a way to iterate over each character already at this point
then, using a method in class Character, you can check if the specific char suits you
and add it to the set